eng/Signing.props

```diff
@@ -1,6 +1,8 @@
 <Project>
   <ItemGroup>
-    <ItemsToSign Include="$(VisualStudioSetupInsertionPath)Microsoft.Build.UnGAC.exe" />
+    <!-- Do not include VS setup insertion files when not building on desktop. The Vsix generation targets do not get imported,
```

Comment:
If this file is still going to be publsihed and not signed, please consider adding this file the https://github.com/dotnet/msbuild/blob/332df0eeee64b9757ffe063d58bff64ded33e05c/.vsts-dotnet.yml#L63 for this particular case.

Reply:
It just ends up in the obj dir, and should not get published.